### ISSUE TITLE
[FEAT]: Subtitles

### ISSUE THREAD

**Issue by TonyH1234** [nova-video-player/aos-AVP#1221]
### Description

Is it possible to include an option to always select an external srt as default subtitle if it is the same name as the video file? I think an older version did this.

**Comment by gkeiko** [comment]
right! hope to improve TKS

**Comment by ralves58** [comment]
I totally agree, it would be an excellent setting, I don't know why it isn't there yet.
The embedded subtitles are not always the best option for a video file...
